code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -13,8 +13,16 @@
 		var/operating = FALSE
 		var/obj/item/weapon/reagent_containers/beaker = null
 		var/limit = 10
-		var/list/blend_items = list (
+		var/static/list/blend_items
+		var/static/list/juice_items
+		var/static/list/dried_items
 
+		var/list/holdingitems = list()
+
+/obj/machinery/reagentgrinder/Initialize()
+	. = ..()
+	beaker = new /obj/item/weapon/reagent_containers/glass/beaker/large(src)
+	blend_items = typecacheof(list(
```

Comment:
as said in IRC, assoc lists can't be typecaches
Amusingly, this object was the reason typecaches were removed from is_type_in_list and made a manual optimisation.